README.md

```diff
@@ -17,13 +17,17 @@
 | [@hop-protocol/frontend](packages/frontend)                   | [![npm version](https://badge.fury.io/js/%40hop-protocol%2Ffrontend.svg)](https://badge.fury.io/js/%40hop-protocol%2Ffrontend)                    | React Frontend UI                           |
 | [@hop-protocol/hop-node](packages/hop-node)                   | [![npm version](https://badge.fury.io/js/%40hop-protocol%2Fhop-node.svg)](https://badge.fury.io/js/%40hop-protocol%2Fhop-node)                    | TypeScript Hop Node                         |
 
-## Getting started
+## Quickstart
 
-```bash
-npm run bootstrap
-cd packages/frontend
-npm run dev
-```
+Install dependencies & link packages
+
+    yarn install
```

Comment:
I noticed there's no backticks to define code block; how does the markdown parser know which syntax highligher to use?

Reply:
if text is indented >= 4 spaces, it automatically becomes a regular code block.

since these are terminal commands, i figured we don't really need syntax highlighting 🤷‍♂️